App.js

```diff
@@ -1,25 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import SvgComponent from "./LogoSvg";
+import LogoSvg from "./LogoSvg";
+import Quotes from "./Quotes";
+import AnimateSvg from './AnimateSvg';
+import AnimateQuotes from './AnimateQuotes';
```

Comment:
Added import statement for animation wrapper for Quotes component.

Reply:
remove this comment